### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## Math magicians: Full Website
This is the Fourth project of the Math Magicians application. I set up the 3 pages using `React Router` and give some styles for the page. The requirements of this project are:

- Using linters to free of errors.
- Using Gitflow
- Documented in professional way
- Create a website consisting of 3 pages: `Home`, `Calculator`, and `Quote`

### CODE REVIEW

File: src/components/CalculatorPage.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import styles from './styles/CalculatorPage.module.css';
+import Calculator from './Calculator';
```
Comment (Lizdev-05) [inline src/components/CalculatorPage.js]: Great job so far 👍. Kindly check the path, 'Calculator.js' does not match the corresponding name on disk: './src/components/calculator.js'.

If you change the "./Calculator" to  "./calculator" as you have it on your disk, your app will work fine. 📘 😊

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Iqbal-Elham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Descriptive Pull Request message ✔️
- [ ] Good use of GitHub flow ✔️
- [ ] Meaningful commit messages ✔️
- [ ] No linter error ✔️
- [ ] Brilliant implementation ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CalculatorPage.js (hunk 1 above)
Comment (Iqbal-Elham) [reply src/components/CalculatorPage.js]: Thanks you so much for the suggestion @Lizdev-05

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Iqbal-Elham , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters
✅ ... Great communication with others

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**